NEWS

```diff
@@ -30,7 +21,12 @@ Features:
 - GPU-to-Infiniband HCA affinity support based on locality/distance (PCIe)
 
 Bugfixes:
-- Multiple bugfixes (full list on github)
+- Fix overflow in RC/DC flush operations
+- Update description in SPEC file and README
+- Fix RoCE source port for dc_mlx5 flow control
+- Improve ucx_info help message
+- Fix segfault in UCP, due to int truncation in count_one_bits()
+- Multiple other bugfixes (full list on github)
```

Comment:
@shamisp This seems relevant here, but you are repeating this again on line 36 (you might want to remove it).

Reply:
@manjugv thanks, copy-n-past error. The one on line 36 has to be removed